testgrid/config/config.yaml

```diff
@@ -2674,78 +2581,35 @@ dashboards:
   - name: periodic-bazel-test-1.7
     test_group_name: periodic-kubernetes-bazel-test-1-7
 
+# These are the release *blocking* tests.  These provide a valid binary signal
+# to gate releases (alpha, beta, official).
+# This list is used by kubernetes/release/lib/releaselib.sh to provide no/nogo
+# signal for releases.
+# The first test/suite in the list is the primary suite for release purposes
 - name: release-1.7-blocking
   dashboard_tab:
-  - name: build-1.7
-    test_group_name: ci-kubernetes-build-1.7
+  - name: gce-release-1-7
+    test_group_name: ci-kubernetes-e2e-gce-release-1-7
+  - name: gce-serial-release-1-7
+    test_group_name: ci-kubernetes-e2e-gce-serial-release-1-7
+  - name: gce-slow-release-1-7
+    test_group_name: ci-kubernetes-e2e-gce-slow-release-1-7
+  - name: gce-reboot-release-1-7
+    test_group_name: ci-kubernetes-e2e-gce-reboot-release-1-7
+  - name: gce-scalability-release-1-7
+    test_group_name: ci-kubernetes-e2e-gce-scalability-release-1-7
   - name: test-go-1.7
     test_group_name: ci-kubernetes-test-go-release-1.7
   - name: verify-1.7
     test_group_name: ci-kubernetes-verify-release-1.7
   - name: kubelet-1.7
     test_group_name: ci-kubernetes-node-kubelet-1.7
-  - name: federation-build-1.7
-    test_group_name: ci-kubernetes-federation-build-1.7
-  - name: gce-release-1-7
-    test_group_name: ci-kubernetes-e2e-gce-release-1-7
-  - name: gci-gce-release-1-7
-    test_group_name: ci-kubernetes-e2e-gci-gce-release-1-7
+  - name: gke-serial-release-1-7
+    test_group_name: ci-kubernetes-e2e-gke-serial-release-1-7
   - name: gke-release-1-7
     test_group_name: ci-kubernetes-e2e-gke-release-1-7
-  - name: gci-gke-release-1-7
-    test_group_name: ci-kubernetes-e2e-gci-gke-release-1-7
-  - name: aws-release-1-7
-    test_group_name: ci-kubernetes-e2e-kops-aws-release-1-7
-  - name: gce-slow-release-1-7
-    test_group_name: ci-kubernetes-e2e-gce-slow-release-1-7
-  - name: gci-gce-slow-release-1-7
-    test_group_name: ci-kubernetes-e2e-gci-gce-slow-release-1-7
   - name: gke-slow-release-1-7
     test_group_name: ci-kubernetes-e2e-gke-slow-release-1-7
-  - name: gci-gke-slow-release-1-7
-    test_group_name: ci-kubernetes-e2e-gci-gke-slow-release-1-7
-  - name: gce-serial-release-1-7
-    test_group_name: ci-kubernetes-e2e-gce-serial-release-1-7
-  - name: gci-gce-serial-release-1-7
-    test_group_name: ci-kubernetes-e2e-gci-gce-serial-release-1-7
-  - name: gke-serial-release-1-7
-    test_group_name: ci-kubernetes-e2e-gke-serial-release-1-7
-  - name: gci-gke-serial-release-1-7
-    test_group_name: ci-kubernetes-e2e-gci-gke-serial-release-1-7
-  - name: gce-federation-release-1-7
```

Comment:
Only keep federation for master and 1.7+. For 1.6 and lower, federation test is permanently broken and WNF.